### DISCUSSION TITLE
Welcome to Quizlet_DUT Discussions!

### DISCUSSION THREAD

**Discussion by TrQuan17** [TrQuan17/Quizlet_DUT#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MarkPage2k1** [comment]
Good job! @TrQuan17 😉😉😉

**Comment by MarkPage2k1** [comment]
Hello friends, I want to join your community to learn the knowledge that I am lacking, as well as help people with the problems that I face and solve them together. 🙂🙂🙂
